dev-environment/mlflow/run.sh

```diff
@@ -9,14 +9,11 @@ sleep 5
 echo "Starting new mlflow run..."
 
 cd $MLFLOW_HOME && \
-mlflow run --no-conda $MLFLOW_WORK_DIR/models/sklearn_elasticnet_wine -P alpha=0.42 > run.out 2>&1 && \
+mlflow run --no-conda $MLFLOW_WORK_DIR/models/sklearn_elasticnet_wine -P alpha=0.42 2>&1 | tee run.out && \
+echo "Mlflow run finished" && \
 RUN_ID="$(cat run.out | tail -n 1 | grep -oP "(?<=').*?(?=')")" && \
-rm -f run.out
-
-echo "Mlflow run finished"
-echo "Mlflow run id: $RUN_ID"
-
-echo "Serving trained model in Mlflow registry on port $MODEL_PORT"
-
+rm -f run.out && \
```

Comment:
why this commands and connected (&&) together?

Reply:
it's a concise way to say "if any of the previous commands failed the next one won't execute and non zero code is returned"

Reply:
oh, fair point